src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -43,12 +44,16 @@ export class LiveAnnouncer implements OnDestroy {
 
   /**
    * Announces a message to screenreaders.
-   * @param message Message to be announced to the screenreader
-   * @param politeness The politeness of the announcer element
+   * @param message Message to be announced to the screenreader.
+   * @param politeness The politeness of the announcer element.
+   * @param duration Time in milliseconds after which to clear out the announcer element.
```

Comment:
I think it would be good to mention that this duration is being taken into account once the `100ms` timeout finished.